README.md

```diff
@@ -218,7 +219,7 @@ To prevent abuse of the API, browsers could maintain a list of allowed or disall
 
 #### Mitigation: Per-Site Issuer Limits
 
-The rate of identity leakage from one site to another increases with the number of tokens redeemed on a page. To avoid abuse, there should be strict limits on the number of token issuers contacted per origin (e.g. 2), and those contacted should be persisted in browser storage to avoid excessively rotating issuers on subsequent visits.
+The rate of identity leakage from one site to another increases with the number of tokens redeemed on a page. To avoid abuse, there should be strict limits on the number of token issuers contacted per origin (e.g. 2); this limit should apply for both issuance and redemption; and the bound issuers should be persisted in browser storage to avoid excessively rotating issuers on subsequent visits.
```

Comment:
s/per origin/per top-level origin/

s/the bound issuers/the site-bound issuers/